[2025-04-23T22:43:13.428+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T22:43:13.443+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_price_prediction.predict manual__2025-04-23T22:41:16.787106+00:00 [queued]>
[2025-04-23T22:43:13.451+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_price_prediction.predict manual__2025-04-23T22:41:16.787106+00:00 [queued]>
[2025-04-23T22:43:13.453+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-04-23T22:43:13.466+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): predict> on 2025-04-23 22:41:16.787106+00:00
[2025-04-23T22:43:13.474+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2081) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-23T22:43:13.477+0000] {standard_task_runner.py:72} INFO - Started process 2084 to run task
[2025-04-23T22:43:13.474+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_price_prediction', 'predict', 'manual__2025-04-23T22:41:16.787106+00:00', '--job-id', '299', '--raw', '--subdir', 'DAGS_FOLDER/226_lab1_train_predict.py', '--cfg-path', '/tmp/tmpza6y981l']
[2025-04-23T22:43:13.479+0000] {standard_task_runner.py:105} INFO - Job 299: Subtask predict
[2025-04-23T22:43:13.525+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_price_prediction.predict manual__2025-04-23T22:41:16.787106+00:00 [running]> on host d01273b1fd04
[2025-04-23T22:43:13.611+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_price_prediction' AIRFLOW_CTX_TASK_ID='predict' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T22:41:16.787106+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T22:41:16.787106+00:00'
[2025-04-23T22:43:13.612+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-23T22:43:13.625+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-23T22:43:13.626+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-04-23T22:43:13.628+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-23T22:43:13.629+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-23T22:43:38.641+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-23T22:43:38.743+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-23T22:43:38.745+0000] {connection.py:788} INFO - closed
[2025-04-23T22:43:38.800+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-04-23T22:43:38.860+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-23T22:43:38.871+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-23T22:43:38.873+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_price_prediction, task_id=predict, run_id=manual__2025-04-23T22:41:16.787106+00:00, execution_date=20250423T224116, start_date=20250423T224313, end_date=20250423T224338
[2025-04-23T22:43:38.903+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-23T22:43:38.917+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T22:43:38.921+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
